applications/luci-app-cloudflared/Makefile

```diff
@@ -9,7 +9,7 @@ LUCI_DEPENDS:=+cloudflared
 LUCI_DESCRIPTION:=LuCI support for Cloudflare Zero Trust Tunnels
 
 PKG_MAINTAINER:=Hilman Maulana <hilman0.0maulana@gmail.com>, Sergey Ponomarev <stokito@gmail.com>
-PKG_VERSION:=1.0
+PKG_VERSION:=1.1
```

Comment:
Just out of curiosity, why would you chang the versions manually instead of just letting LuCI do it automatically?

Reply:
This is probably makes sense for code Luci modules and it will set a version from git.
Here we have an own semantic version.